modules/serving-transport-encryption-configuring.adoc

```diff
@@ -0,0 +1,103 @@
+:_content-type: PROCEDURE
+[id="serving-transport-encryption-configuring_{context}"]
+= Configuring transport encryption
+
+Configuring transport encryption consists of two parts:
+
+. Specifying ClusterIssuer to use:
+
+`clusterLocalIssuerRef`: issuer for cluster-local-domain certificates used for ingress.
+
+`systemInternalIssuerRef`: issuer for certificates for system-internal-tls certificates used by Knative internal components.
+
+. Specifying transport encryption features to use:
```

Comment:
Not sure if this will be different in the final docs but the rendered start counting the bullets from 1 again. I believe this bullet should have number `2.`.  Screenshot:
![image](https://github.com/user-attachments/assets/6dce74a1-7a31-439e-85e5-407875c99207)

Reply:
Fixed the markup.